go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/vishvananda/netlink
+
+go 1.12
+
+require (
+	github.com/vishvananda/netns v0.0.0-20180720170159-13995c7128cc
```

Comment:
If this PR looks good to you @vishvananda @aboch, it's probably nice to have https://github.com/vishvananda/netns/pull/35 merged before this one (and, if possible, tag a release for https://github.com/vishvananda/netns)

Reply:
@tklauser the netns PR is merged. You may want to update this reference then.

Reply:
Done, thanks.